### ISSUE TITLE
Relate to Wikidata

### ISSUE THREAD

**Issue by ppKrauss** [OpenGovLD/skos#10]
Some terms have been developed at Wikipedia, and ensures good semantic reference in the form of Wikidata-ID. Some examples: 
- "city council" https://www.wikidata.org/wiki/Q3154693
- "Committee on Finance"  https://www.wikidata.org/wiki/Q5153053
- ...

PS: [similar discussion here](https://github.com/schemaorg/schemaorg/issues/280).

**Comment by akuckartz** [comment]
Yes. I agree that aligment makes sense. I am very interested in avoiding information islands.

**Comment by ppKrauss** [comment]
Perhaps with Wikidata we can add to "avoiding information islands" a kind of "facilitate translations" ;-)

Can I try to add Portuguese (_pt_)?

---

If you agree to advance in a  next step (hands-on!)...   Do you have an SKOS-graph example with this relation?  In the [terms.jsonld](https://github.com/OpenGovLD/skos/blob/master/terms.jsonld) file, the graph nodes are "skos:Concept", so need to add to each (when possible and make sense) a  "owl:equivalentProperty" or "skos:exactMatch", or  "skos:closeMatch"... I not know how to manage SKOS-jsonld representations.

**Comment by akuckartz** [comment]
@ppKrauss PRs requests are welcome. I am still reviewing tools to better manage the `skos:Concept`s.

**Comment by akuckartz** [comment]
I just added "Finanzausschuss" on this page: https://www.wikidata.org/wiki/Q5153053

That was very easy. But I wonder about the editorial workflow.

**Comment by akuckartz** [comment]
As a small experiment I just added the German "Finanzausschuss" on the page for "Committee on Finance" (https://www.wikidata.org/wiki/Q5153053).

That was very easy. But I wonder about the editorial workflow.

How can it be expressed that a "Committee on Finance" is a "Committee" ?

**Comment by ppKrauss** [comment]
@akuckartz  Yes,  using [Property:P279](https://www.wikidata.org/wiki/Property:P279) that is the *wd:subclassOf* property, any one can edit adding the "subclass of [committe](https://www.wikidata.org/wiki/Q865588)" statment, as I do at [wd:CommitteeOnFinance](https://www.wikidata.org/wiki/Q5153053) (Q5153053).

We can map both:

* from Wikidata to *external vocabularies* (popular ones as schemaOrg) 
* from your vocabulary (here) to Wikidata
